docusaurus/docs/operate/configs/relayminer_config.md

```diff
@@ -104,42 +111,59 @@ and `supplier` specific sections and configurations.
 ## Global options
 
 ```yaml
-signing_key_name: <string>
+default_signing_key_names: [ <string>, <string> ]
 smt_store_path: <string>
 ```
 
-### `signing_key_name`
+### `default_signing_key_names`
 
-_`Required`_
+_`Required`_ if `suppliers.*.signing_key_names` is not specified. 
+
+This configuration option specifies a list of key names from the keyring that the `RelayMiner` will use to sign transactions. These keys are used to derive the public key and the corresponding address, which are essential for interacting with the Pocket network. Each key name listed here must be present in the keyring used to start the `RelayMiner` instance.
+
+#### Example Configuration
+```yaml
+default_signing_key_names:
+  - keyname1
+  - keyname2
+```
 
-The name of the key that will be used to sign transactions, derive the public key
-and the corresponding address. This key name MUST be present in the keyring that is used
-to start the `RelayMiner` instance.
+In this example, `keyname1` and `keyname2` are the names of keys in the keyring that will be used to sign transactions.
+
+#### How It Works
+
+- If a supplier does not specify its own `signing_key_names`, the supplier will use the keys listed in `default_signing_key_names`.
```

Comment:
Consider adding a flow chart to make this visual: https://mermaid.js.org/syntax/flowchart.html

I bet you can ask ChatGPT to create one based on the text as a starting point.

Reply:
I've restructured the documentation page - I think it is more user-friendly now. Also includes the diagram.